cocos2d/core/assets/CCAudioClip.js

```diff
@@ -127,6 +127,7 @@ var AudioClip = cc.Class({
                 let self = this;
                 cc.assetManager.postLoadNative(this, function (err) {
                     self._loading = false;
+                    self.loaded = true;
```

Comment:
Need to check the value of err, if err has a value it means an error occurred, probably due to a problem with resource loading.

Reply:
Added code to check the err parameter